swiftspec/core.py

```diff
@@ -142,18 +142,34 @@ def get_tokens_from_env(self):
         token = os.environ.get("OS_AUTH_TOKEN")
         url = os.environ.get("OS_STORAGE_URL")
         if token and url:
-            return [{"token": token, "url": url}]
+            return [{"X-Auth-Token": token, "url": url}]
+
+        sig = os.environ.get("TEMP_URL_SIG")
+        expire = os.environ.get("TEMP_URL_EXPIRES")
+        prf = os.environ.get("TEMP_URL_PREFIX")
+
+        if url and sig and expire:
+            return [{"url":url,"temp_url_sig": sig, "temp_url_expires": expire, "temp_url_prefix":prf}]
```

Comment:
According to the [docs](https://docs.openstack.org/swift/latest/api/temporary_url_middleware.html), `temp_url_prefix` is optional. Thus maybe it should be optional in here as well?